packages/wrangler/templates/middleware/loader-sw.ts

```diff
@@ -183,7 +183,7 @@ __facade__originalAddEventListener__("fetch", (event) => {
 			});
 
 			__FACADE_EVENT_TARGET__.dispatchEvent(facadeEvent);
-			event.waitUntil(Promise.all(facadeEvent[__facade_waitUntil__]));
+			event.waitUntil(void Promise.all(facadeEvent[__facade_waitUntil__]));
```

Comment:
This is needed because we improved the typing for `waitUntil` in `workers-types` v4